From 939d6dd50a2d2a96de4735917dd42c2c508d67cc Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Thu, 4 Jan 2024 07:54:42 +0100
Subject: [PATCH] hyprlang.pc.in: fix include path

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 hyprlang.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hyprlang.pc.in b/hyprlang.pc.in
index 97350dc..d09a8cf 100644
--- a/hyprlang.pc.in
+++ b/hyprlang.pc.in
@@ -1,5 +1,5 @@
 prefix=@PREFIX@
-includedir=@INCLUDE@
+includedir=${prefix}/include
 libdir=${prefix}/lib
 
 Name: hyprlang
-- 
2.43.0

