From a8fa47e90bddccfe6dd9f267dd7a189793c8adca Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Mon, 2 Aug 2021 06:59:38 +0200
Subject: [PATCH] meson.build: fix scdoc not to be found

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 180f0fb..0973213 100644
--- a/meson.build
+++ b/meson.build
@@ -130,7 +130,7 @@ configure_file(
 	install_dir: datadir + '/dbus-1/services',
 )
 
-scdoc = dependency('scdoc', required: get_option('man-pages'), version: '>= 1.9.7')
+scdoc = dependency('scdoc', native: true, required: get_option('man-pages'), version: '>= 1.9.7') 
 
 if scdoc.found()
 	sh = find_program('sh')
-- 
2.25.1

