From bd5d23cc82e9f7c065571b6695fe024c6f51051e Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sun, 6 Jun 2021 18:25:08 +0200
Subject: [PATCH] meson.build: avoid scdoc dependency

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 50cc9eb..993b7fd 100644
--- a/meson.build
+++ b/meson.build
@@ -220,14 +220,13 @@ endif
 
 install_data(
   'LICENSE', 'README.md', 'CHANGELOG.md',
-  install_dir: join_paths(get_option('datadir'), 'doc', 'foot'))
+  install_dir: join_paths(get_option('datadir'), 'foot'))
 install_data(
   'foot.desktop', 'foot-server.desktop',
   install_dir: join_paths(get_option('datadir'), 'applications'))
 install_data('foot.ini', install_dir: join_paths(get_option('datadir'), 'foot'))
 
 subdir('completions')
-subdir('doc')
 subdir('icons')
 
 summary(
-- 
2.25.1

