From f14a8d54650f40ecb6f018fd7d5ac7cffbc9358f Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sun, 7 Aug 2022 17:03:05 +0200
Subject: [PATCH] meson.build: build with system wlroots

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 meson.build | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/meson.build b/meson.build
index dfcb0e1..42a90de 100644
--- a/meson.build
+++ b/meson.build
@@ -19,14 +19,14 @@ add_project_arguments(
 
 cc = meson.get_compiler('c')
 
-wlroots_subproj = subproject(
-  'wlroots',
-  default_options: ['examples=false', 'default_library=static'],
-  version: '>= 0.11.0',
-  required: true,
-)
-
-wlroots        = wlroots_subproj.get_variable('wlroots')
+#wlroots_subproj = subproject(
+#  'wlroots',
+#  default_options: ['examples=false', 'default_library=static'],
+#  version: '>= 0.11.0',
+#  required: true,
+#)
+
+wlroots        = dependency('wlroots', version: '>= 0.11.0')
 wayland_protos = dependency('wayland-protocols', version: '>=1.18')
 wayland_server = dependency('wayland-server')
 wayland_client = dependency('wayland-client')
-- 
2.34.1

