From 8b19cc784d4f0b56e45e22c911b7f9671ae92ca4 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sun, 6 Jun 2021 18:23:06 +0200
Subject: [PATCH] meson.build: avoid scdoc dependency

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 meson.build | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 11ac4cb..8a14738 100644
--- a/meson.build
+++ b/meson.build
@@ -123,9 +123,8 @@ if not meson.is_subproject()
 
   install_data(
     'LICENSE', 'README.md', 'CHANGELOG.md',
-    install_dir: join_paths(get_option('datadir'), 'doc', 'fcft'))
+    install_dir: join_paths(get_option('datadir'), 'fcft'))
 
-  subdir('doc')
 endif
 
 summary(
-- 
2.25.1

