From 3fcdc43e301b5f166619689caac4751b601faf08 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Tue, 15 Feb 2022 01:03:03 +0100
Subject: [PATCH] meson.build: add missing header

---
 meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meson.build b/meson.build
index 43b60e8..94c9f84 100644
--- a/meson.build
+++ b/meson.build
@@ -76,6 +76,7 @@ headers_parser = [
 'wayfire/parser/action_parser.hpp',
 'wayfire/parser/condition_parser.hpp',
 'wayfire/parser/rule_parser.hpp',
+'wayfire/parser/lambda_rule_parser.hpp',
 ]
 
 headers_rule = [
-- 
2.25.1

